app/retail/emails.py

```diff
@@ -487,41 +500,41 @@ def render_new_bounty_roundup(to_email):
 '''
     highlights = [
         {
-            'who': 'travisdmathis',
+            'who': 'cryptomental',
             'who_link': True,
-            'what': 'The largest Gitcoin bounty ever completed! Integrated Balance and Dharma in a big way.',
-            'link': 'https://gitcoin.co/issue/balance-io/balance-manager/195/553',
+            'what': 'Set up Travis and Testing with Truffle + Ethereum-Bridge on MARKET Protocol.',
+            'link': 'https://gitcoin.co/issue/MARKETProtocol/MARKET.js/33/656',
             'link_copy': 'See more',
         },
         {
-            'who': 'bradysheridan',
+            'who': 'scsaba',
             'who_link': True,
-            'what': 'Integrated Netlify CMS and created the new MARKET Protocol blog!',
-            'link': 'https://gitcoin.co/issue/MARKETProtocol/website/136/581',
+            'what': 'Made possible to view address of an added token on MetaMask!',
+            'link': 'https://gitcoin.co/issue/MetaMask/metamask-extension/4440/644',
             'link_copy': 'View more',
         },
         {
-            'who': 'IRus',
+            'who': 'justpixel',
             'who_link': True,
-            'what': 'Made CircleCI builds cacheable for Cyber Congress.',
-            'link': 'https://gitcoin.co/issue/cybercongress/cyber-search/184/577',
+            'what': 'Created a great animation of the Grow Open Source tree!'
```

Comment:
Missing a comma at the end of this dictionary value. `'what': 'Created a great animation of the Grow Open Source tree!',`